app/routes/internshipProjects/$projectId/index.tsx

```diff
@@ -77,22 +88,20 @@ export default function ProjectDetail() {
           <p className="descriptionProposal">{projects.description}</p>
           {user && (
             <Grid style={{ position: "absolute", top: 0, right: 0 }}>
-              <Form
-              method="put"
-              action='./appliedproject'
-              >
-              <Button
-                className="contained"
-                type='submit'
-                sx={{
-                  width: "200px",
-                  height: "40px",
-                  fontSize: "1em",
-                  margin: 2,
-                }}
-                disabled={appliedProjects.includes(projects.name)}
+              <Form method="put" action='./appliedproject'>
+                <Button
+                  className="contained"
+                  type='submit'
+                  sx={{
+                    width: "200px",
+                    height: "40px",
+                    fontSize: "1em",
+                    margin: 2,
+                  }}
+                  onClick={handleApply} 
+                  disabled={fetcher.state === 'loading' || appliedProjects.includes(projects.name)}
               >
-                APPLY
+                {appliedProjects.includes(projects.name) ? 'APPLIED' : 'APPLY'}
```

Comment:
This line does not seem to be working, at least for me. It stays as App
<img width="1196" alt="Screenshot 2023-11-13 at 13 35 28" src="https://github.com/wizeline/remix-project-lab/assets/149706555/9846cdaf-08c7-4a7b-99ab-7432db0f8052">
ly

Reply:
Also, I wonder if we could change the style of the button when disabled as it is hard to read the black text on the dark red background. Maybe we could leave it as white but add a blocked cursor instead?

Reply:
Thanks @beatriz , I'll check it out 👍

Reply:
Working like a charm now :)